packages/api/src/app-router.ts

```diff
@@ -197,7 +198,9 @@ export default async function makeApp(params: CliArgs) {
 
   if (returnRegionInOrchestrator) {
     app.use((req, res, next) => {
-      req.orchestratorsGetters.push(regionsGetter);
+      req.orchestratorsGetters.push(
+        regionsGetter(halfRegionOrchestratorsUntrusted)
```

Comment:
Contextualizing the other comment, if regionsGetter is made to return the orchestrators directly instead of a function, here you could have:
`() => regionsGetter(halfRegionO...)`

Then this function indirection stays localized here, which is where we actually need it. Other places can just use regionGetter (like the API controllers are using now)

Reply:
Awesome, that's what I was looking for thanks! Much nicer to curry it at the calling site and keep the actual function clean